### ISSUE TITLE
update() should trigger callbacks

### ISSUE THREAD

**Issue by mchccc** [odyniec/imgareaselect#20]
As the title says, I think that a call to the update() function should trigger most of the callbacks available..
What do you think?

**Comment by drewish** [comment]
Yeah I agree. I'm banging my head against the fact that events don't fire if you assign a selection in the initial imgAreaSelect() call. I was hoping I could just call update() afterwards and force it.

**Comment by acrispino** [comment]
This would also be very convenient.

Similarly it would be nice to also trigger callbacks if initial selection dimensions are given to imgAreaSelect()

**Comment by tonytopper** [comment]
Yeah, I am currently searching for a graceful way to trigger onSelectChange when I call setSelection from an instance of imgAreaSelect.  My expectation was that calling setSelection would do this.
